templates/terraform/custom_flatten/data_catalog_tag.go.erb

```diff
@@ -0,0 +1,53 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	if v == nil {
+		return v
+	}
+	l := v.(map[string]interface{})
+	transformed := make([]interface{}, 0, len(l))
+	for k, raw := range l {
```

Comment:
Can you add a comment on what this is doing that the generated flattener doesn't?

Reply:
@danawillow - thanks for mentioning this, i looked further into it and although the documentation has the fields snake case (which i originally wrote them as), i tried them with camel case and it worked, so i switched them there. that's originally what the custom flattener was for. i repurposed it now specifically for `enumValue` because without the custom flattener it returns an array of interfaces rather than the string it's expecting. hope that makes sense!